Applications/Games/Subnautica/Steam/script.js

```diff
@@ -0,0 +1,31 @@
+include(["Engines", "Wine", "QuickScript", "SteamScript"]);
+include(["Engines", "Wine", "Verbs", "vcrun2013"]);
+include(["Engines", "Wine", "Verbs", "vcrun2008"]);
+include(["Engines", "Wine", "Verbs", "d3dx9_42"]);
+include(["Engines", "Wine", "Verbs", "xact"]);
+include(["Engines", "Wine", "Verbs", "corefonts"]);
+
+var screenSize = java.awt.Toolkit.getDefaultToolkit().getScreenSize();
+wine.setVirtualDesktop(screenSize.width, screenSize.height);
```

Comment:
Have you tested this? I don't think it can work. `wine` should be undefined at this point.